jgrapht-ext/src/main/java/org/jgrapht/ext/DSVUtils.java

```diff
@@ -0,0 +1,138 @@
+/* ==========================================
+ * JGraphT : a free Java graph-theory library
+ * ==========================================
+ *
+ * Project Info:  http://jgrapht.sourceforge.net/
+ * Project Creator:  Barak Naveh (http://sourceforge.net/users/barak_naveh)
+ *
+ * (C) Copyright 2003-2008, by Barak Naveh and Contributors.
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+/* ------------------
+ * DSVUtils.java
+ * ------------------
+ * (C) Copyright 2016, by Dimitrios Michail and Contributors.
+ *
+ * Original Author:  Dimitrios Michail
+ * Contributors: -
+ *
+ * Changes
+ * -------
+ * 20-Aug-2016 : Initial Version (DM);
+ *
+ */
+package org.jgrapht.ext;
+
+/**
+ * Helper utilities for Delimiter-separated values.
+ * 
+ * @author Dimitrios Michail
+ * @since August 2016
+ */
+class DSVUtils
+{
+    private static final char DSV_QUOTE = '"';
+    private static final char DSV_LF = '\n';
+    private static final char DSV_CR = '\r';
+    private static final String DSV_QUOTE_AS_STRING = String.valueOf(DSV_QUOTE);
+
+    /**
+     * Test if a character can be used as a delimiter in a Delimiter-separated
+     * values file.
+     * 
+     * @param delimiter the character to test
+     * @return {@code true} if the character can be used as a delimiter,
+     *         {@code} false otherwise
+     */
+    public static boolean isValidDelimiter(char delimiter)
+    {
+        return delimiter != DSV_LF && delimiter != DSV_CR
+            && delimiter != DSV_QUOTE;
+    }
+
+    /**
+     * Escape a Delimiter-separated values string.
+     * 
```

Comment:
Since this is a public class with static functions as opposed to a local package level helper class, it may be useful to add a bit more details as to what it means to escape a delimiter-separated values string. (My first interpretation was to add a '\' in front of every special character except the delimiter).